auth/oauth.py

```diff
@@ -27,6 +37,63 @@ def validate_bearer_auth(auth_header):
 
 
 def validate_oauth_token(token):
+    if is_jwt(token):
+        return validate_sso_oauth_token(token)
+    else:
+        return validate_app_oauth_token(token)
+
+
+def validate_sso_oauth_token(token):
```

Comment:
Is this different from an oauth token we would normally validate?

Reply:
yes our oauth token has a different format that a standard jwt token